dressup/src/pages/Homepage/TopBrandsSection/Topbrandssection.js

```diff
@@ -0,0 +1,27 @@
+import "./Topbrandssection.css";
+
+import img6 from "../../../images/nike.jpeg";
+import img7 from "../../../images/levis.png";
+import img8 from "../../../images/Hm.png";
+import img9 from "../../../images/hrx.png";
+import img10 from "../../../images/anouk.jpeg";
+import img11 from "../../../images/only.png";
+import img12 from "../../../images/gap.jpeg";
+import img13 from "../../../images/mac.png";
```

Comment:
suggestion: instead of importing all images line by line here, create index.js file in images and export all images from index.js, so importing images in this file become easy.